### PR TITLE
SUR-255 Implement Skeleton component

### PR DESCRIPTION
**Pull request by JelenaTakac**
### Description

# Skeleton Component Documentation

## Description

The `Skeleton` component can be used as an alternative loading indicator to the spinner by mimicking the content that will be loaded such as text, images, or video.

### `variant`
- **Type:** `string`
- **Default:** `"rectengular"`
- **Description:** Defines the style variant of the skeleton. Options include:
  - `"rectengular"` 
  - `"circular"`

### `className`
- **Type:** `string`
- **Description:** Additional custom classes to be added to the skeleton component, allowing for further customization with Tailwind CSS.

### Basic Example

```jsx
import { Skeleton } from '@bsf/force-ui';

const App = () => (
    <div>
        {/* Rectangular skeleton representing a text block */}
        <Skeleton variant="rectangular" className="w-full h-4" />
        {/* Circular skeleton representing an avatar */}
        <Skeleton variant="circular" className="w-10 h-10 " />
        {/* Rectangular skeleton representing a image or video*/}
        <Skeleton variant="rectangular" className="w-64 h-12" />
    </div>
);

export default App;
```

### Checklist:

-   [ ] My code is tested
-   [ ] My code passes the PHPCS tests
-   [ ] I've created the npm build.
-   [ ] My code follows accessibility standards 
-   [ ] My code has proper inline documentation 
-   [ ] I've included any necessary tests 
-   [ ] I've included developer documentation 
-   [ ] I've added proper labels to this pull request

### CODE REVIEW

File: src/components/skeleton/skeleton.jsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { cn } from '@/utilities/functions';
+
+const Skeleton = ( {
+	variant = 'rectengular', // rectengular, circular
```
Comment (vrundakansara) [inline src/components/skeleton/skeleton.jsx:5]:
```suggestion
	variant = 'rectangular', // rectangular, circular
```

**Review by vrundakansara — CHANGES_REQUESTED** [review]

File: src/components/skeleton/readme.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+## Description
+
+The `Skeleton` component can be used as an alternative loading indicator to the spinner by mimicking the content that will be loaded such as text, images, or video.
+
+## Props
+
+### `variant`
+- **Type:** `string`
+- **Default:** `"rectengular"`
```
Comment (vrundakansara) [inline src/components/skeleton/readme.md:11]:
```suggestion
- **Default:** `"rectangular"`
```

File: src/components/skeleton/readme.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+- **Description:** Defines the style variant of the skeleton. Options include:
+  - `"rectengular"`
```
Comment (vrundakansara) [inline src/components/skeleton/readme.md:13]:
```suggestion
  - `"rectangular"` 
```

**Review by vrundakansara — APPROVED** [review]

**Review by ravindra114 — CHANGES_REQUESTED** [review]
@JelenaTakac when we not passing prop `variant` it is not picking default rectangular
<Skeleton className="w-full h-6" />
image

Fix - There is a spelling mistake where you set default value of variant in component.

**Pull request merged**